.codebuild/buildspec.yml

```diff
@@ -59,6 +59,7 @@ env:
     testnetindia_PUSH_NOTIFICATION_ENABLED: "true"
     testnetindia_PUSH_ALLOWED_PROVIDERS: "android,ios"
     testnetindia_APPLICATION_NAME: "wallet-service-testnet-india"
+    testnetindia_SERVERLESS_DEPLOY_PREFIX: "wallet-service"
```

Comment:
question(non-blocking): Just double-checking that we won't be using the default `'hathor-wallet-service'` prefix for `testnet-india`, right?